docs/products/metrics/howto/download-influxdb-data.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Download Aiven for InfluxDB data backup
+sidebar_label: Download InfluxDB data
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+
+Download a backup of your Aiven for InfluxDB service data by requesting a backup. This creates a snapshot of your latest data, ready for download.
+
+## Prerequisites
+
+Before you download a backup, make sure you have the following:
+
+- [Aiven API token](/docs/platform/howto/create_authentication_token) with the necessary
+  permissions
+- [Aiven CLI](/docs/tools/cli)
+
+If your InfluxDB service is in a Virtual Private Cloud (VPC) or if you need specific
+network access to your backup, confirm that the correct access settings are enabled.
+Based on your network setup, enable one of the following settings:
+
+- `public_access.user_backup`: Allows access to backups from the public internet
+- `private_access.user_backup`: Allows access to backups within a private network
+- `privatelink_access.user_backup`: Allows access to backups via a private link
+
+To configure access, go to the service's <ConsoleLabel name="overview" /> page,
```

Comment:
Wrong path. It is "Service settings" -> "Cloud and network" -> "three dots" -> "More network configurations".

Reply:
My bad! Thanks for the correction here.